file_server.rst

```diff
@@ -103,42 +207,86 @@ named "Restored folder". This folder is readable by everyone, while its
+===============================
 
-First of all, apply the procedure described in
+First, follow the procedure described in
 :ref:`application_restore-section` by selecting the backup of the **Samba
 module**.
 
-If the restored controller is the first of the domain there are
+After the restoration process completes, further actions may be needed to
+start the file server, depending on the original Samba role: member or
+controller.
+
+Restore a domain member
+-----------------------
+
+To complete the restoration of a **domain member**, navigate to the Samba
+application ``Status`` page.
+
+- If the original IP address and user domain were found, the restore
+  procedure automatically starts the file server. No manual operations are
+  needed.
+
+- Otherwise, the first-configuration procedure will start to acquire the
+  missing information. In this case, a new computer account is created in
+  Active Directory. Follow the procedure as described in
+  :ref:`samba-configuration`.
+
+  If you manually remove the original computer account, you may set the
+  original name as the ``File server alias`` to provide seamless access to
+  shared folders from network clients.
+
+  To manage computer accounts of an NS8 internal Active Directory domain,
+  invoke the ``samba-tool`` command from an NS8 node that hosts a Samba
+  Domain Controller. For example, this command prints an inline help
+  message: ::
+
+    runagent -m samba0 podman exec -ti samba-dc samba-tool computer
+
+  Replace ``samba0`` with your correct DC module identifier.
+
+
+Restore a domain controller
+---------------------------
+
+If the restored **domain controller** is the first in the domain, there are
 two alternatives:
 
-1. If the system IP address is the same used in the backup set, DC
+1. If the system IP address is the same as the one used in the backup set, DC
    services are started automatically and no further actions are required.
 
 2. If the previous condition does not apply, DC services are started using
-   the system VPN IP address as fallback. A similar command is required to
-   select another IP address in a second time: ::
+   the system VPN IP address as a fallback. A similar command is required to
+   select another IP address at a later time: ::
 
      api-cli run module/samba0/set-ipaddress --data '{"ipaddress": "10.15.21.100"}'
 
-   Replace `samba0` with the correct module identifier. Replace the
-   `ipaddress` value with the correct IP address.
+   Replace ``samba0`` with the correct module identifier. Replace the
+   ``ipaddress`` value with the correct IP address.
 
 Otherwise, if one or more domain controllers already exist:
 
-- go to ``Domain and users`` page and open the :guilabel:`Unconfigured
-  provider` link;
-
-- fill the form to join a new DC to the domain.
+- Go to the ``Domain and users`` page and open the :guilabel:`Unconfigured
+  provider` link.
 
+- Fill the form to join a new DC to the domain.
 
 .. rubric:: Footnotes
 
```

Comment:
Just a little correction:
"The guest access in SMB2 and SMB3 is disabled by default in Windows, for more information see microsoft link"